### PR TITLE
Feature/first step basic appservice

### LINKED ISSUE #3: Need appGatewayListenerCertificate into Deploy the infrastructure steps
If the "appGatewayListenerCertificate" is not listed in the parameters.json, the creation of the Application Gateway listener will fail. Therefore, I'd like you to add the "appGatewayListenerCertificate" to the instructions. Furthermore, I believe it would be helpful if you could also include a guide on how to create the certificate.

### PR DESCRIPTION
**Pull request by v-fearam**
## Purpose
Moving Sample application to its own repository
Improve readme file to make it easier to deploy and fix some issues in the instructions.
It includes https://github.com/Azure-Samples/app-service-baseline-implementation/pull/4. After merging this we could close that PR.
Fix: https://github.com/Azure-Samples/app-service-baseline-implementation/issues/3

## Does this introduce a breaking change?

```
[ x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ x] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Complete deploy end to end, following readme instructions. It includes validations.

### CODE REVIEW

**Comment by v-fearam** [comment]
#assign-reviewer:josevarela

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The following are prerequisites.
 ## Prerequisites
 
 1. Ensure you have an [Azure Account](https://azure.microsoft.com/free/)
+1. It is needed an user with [roles](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles) with enough permission to do role assigments. For example User Access Administrator or Owner.
```
Comment (josevarela) [inline README.md]:
```suggestion
1. The deployment must be started by a user who has sufficient permissions to assign [roles](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles), such as a User Access Administrator or Owner.
```

File: README.md [hunk 2]
```diff
@@ -20,44 +21,78 @@ Use the following to deploy the infrastructure.
...
+   - You choose a valid resource group name
 
-    az deployment group create --template-file ./infra-as-code/bicep/main.bicep \
-      --resource-group $RESOURCE_GROUP \
-      --parameters @./infra-as-code/bicep/parameters.json \
-      --parameters baseName=$BASE_NAME
-    ```
+```bash
+   LOCATION=westus3
+   BASE_NAME=<base-resource-name between 3 and 6 charcters>
```
Comment (josevarela) [inline README.md]:
```suggestion
   BASE_NAME=<base-resource-name (between 3 and 6 characters)>

```

File: README.md [hunk 3]
```diff
@@ -84,18 +119,26 @@ Because we have not implemented a CI/CD pipeline with a self-hosted agent, we ne
 1. The deployed storage account does not allow public access, so you will need to temporarily allow access public access from your IP address.
 1. You need to give your user permissions to upload a blob to the storage account.
 
+First, we need to clone the [Simple Web App workload repository](https://github.com/Azure-Samples/app-service-sample-workload)
+
+```bash
+cd ..
+clone https://github.com/Azure-Samples/app-service-sample-workload.git
```
Comment (josevarela) [inline README.md]: git clone(?)

```suggestion
git clone https://github.com/Azure-Samples/app-service-sample-workload.git

```

**Review by josevarela — CHANGES_REQUESTED** [review]
Looks good. A couple of suggestions in-line

File: README.md [hunk 4]
```diff
@@ -86,37 +133,72 @@ Because we have not implemented a CI/CD pipeline with a self-hosted agent, we ne
...
+- Tell the web app to restart
 
 ```bash
-CLIENT_IP_ADDRESS=<your-ip-address>
+CLIENT_IP_ADDRESS=<your-public-ip-address>
 
-STORAGE_ACCOUNT_PREFIX=stg
+STORAGE_ACCOUNT_PREFIX=st
 WEB_APP_PREFIX=app-
 NAME_OF_WEST_STORAGE_ACCOUNT="$STORAGE_ACCOUNT_PREFIX$BASE_NAME"
```
Comment (josevarela) [inline README.md]: Can we rename this variable to remove WEST from the name? Maybe just NAME_OF_STORAGE_ACCOUNT

**Review by josevarela — CHANGES_REQUESTED** [review]
Looks good. One small suggestion to rename a variable in-line

File: README.md (hunk 4 above)
Comment (v-fearam) [reply README.md]: done

**Review by josevarela — APPROVED** [review]
Looks good!

**Pull request merged**